static/app/utils/fields/index.ts

```diff
@@ -308,128 +310,443 @@ export interface FieldDefinition {
+    parameters: [
+      {
+        name: 'column',
+        kind: 'column',
+        columnTypes: validateAndDenyListColumns(
+          [FieldValueType.STRING, FieldValueType.NUMBER, FieldValueType.DURATION],
+          ['id', 'issue', 'user.display']
+        ),
+        defaultValue: 'transaction.duration',
+        required: true,
+      },
+      {
+        name: 'value',
+        kind: 'value',
+        dataType: FieldValueType.STRING,
+        defaultValue: CONDITIONS_ARGUMENTS[0].value,
+        options: CONDITIONS_ARGUMENTS,
+        required: true,
+      },
+      {
+        name: 'value',
+        kind: 'value',
+        dataType: FieldValueType.STRING,
+        defaultValue: '300',
+        required: true,
+      },
+    ],
   },
   [AggregationKey.COUNT_WEB_VITALS]: {
     desc: t('Count of web vitals with a specific status'),
     kind: FieldKind.FUNCTION,
     valueType: FieldValueType.NUMBER,
+    parameters: [
+      {
+        name: 'column',
+        kind: 'column',
+        columnTypes: function ({key}): boolean {
+          return [
+            WebVital.LCP,
+            WebVital.FP,
+            WebVital.FCP,
+            WebVital.FID,
+            WebVital.CLS,
+          ].includes(key as WebVital);
+        },
+        defaultValue: WebVital.LCP,
+        required: true,
+      },
+      {
+        name: 'value',
+        kind: 'value',
+        options: WEB_VITALS_QUALITY,
+        dataType: FieldValueType.STRING,
+        defaultValue: WEB_VITALS_QUALITY[0].value,
+        required: true,
+      },
+    ],
   },
   [AggregationKey.EPS]: {
     desc: t('Events per second'),
     kind: FieldKind.FUNCTION,
     valueType: FieldValueType.NUMBER,
+    parameters: [],
   },
   [AggregationKey.EPM]: {
     desc: t('Events per minute'),
     kind: FieldKind.FUNCTION,
     valueType: FieldValueType.NUMBER,
+    parameters: [],
   },
   [AggregationKey.FAILURE_RATE]: {
     desc: t('Failed event percentage based on transaction.status'),
     kind: FieldKind.FUNCTION,
     valueType: FieldValueType.PERCENTAGE,
+    parameters: [],
   },
   [AggregationKey.FAILURE_COUNT]: {
     desc: t('Failed event count based on transaction.status'),
     kind: FieldKind.FUNCTION,
     valueType: FieldValueType.NUMBER,
+    parameters: [],
   },
   [AggregationKey.MIN]: {
     desc: t('Returns the minimum value of the selected field'),
     kind: FieldKind.FUNCTION,
     valueType: null,
+    parameters: [
+      {
+        name: 'column',
+        kind: 'column',
+        columnTypes: validateForNumericAggregate([
+          FieldValueType.INTEGER,
+          FieldValueType.NUMBER,
+          FieldValueType.DURATION,
+          FieldValueType.DATE,
+          FieldValueType.PERCENTAGE,
+        ]),
+        defaultValue: 'transaction.duration',
+        required: true,
+      },
+    ],
   },
   [AggregationKey.MAX]: {
     desc: t('Returns maximum value of the selected field'),
     kind: FieldKind.FUNCTION,
     valueType: null,
```

Comment:
Should the valueType of this (and other similar functions) be number? I'm seeing a problem when I try to use it (probably because it defaults to string?):

![CleanShot 2024-08-14 at 11 41 03](https://github.com/user-attachments/assets/ed6e72a7-4713-4f15-afbb-474f79eb1329)

Reply:
Yup this makes sense. Lets set valuetype to number.

Reply:
Actually @malwilley, it's a bit odd to be setting valueType to Number here:
<img width="415" alt="Screenshot 2024-08-14 at 4 16 03 PM" src="https://github.com/user-attachments/assets/dac6c9e9-e1fb-409e-b243-04685a5df2b0">

If we were able to provide an array of possible valueTypes and show all the options, or get valueType from the column selected as the parameter it would be ideal.

Reply:
The only option now is to pass valueType as `null` and it defaults to string.

Reply:
Ah, I see. So the value should be dependent on the selected column?

I will follow up by adding support for this. I think I'll need to accept something like:

```
dynamicValueType: (parameters: string[]) => ValueType
```

If I add the above, would it be sufficient?

Reply:
Yeah I think something like that can work or maybe a `isDynamicValueType` boolean that looks through columns provided in parameters and infers the type for the value.